src/aes_hash.rs

```diff
@@ -264,11 +266,13 @@ impl Hasher for AHasherU64 {
     }
 }
 
-#[cfg(feature = "specialize")]
-pub(crate) struct AHasherFixed(pub AHasher);
-
 /// A specialized hasher for fixed size primitives larger than 64 bits.
```

Comment:
This doesn't have to be strictly primitives, just fixed in size. So for example a `(u128, u128)` would be ok with this. It just hasn't been used that way now as I don't have a way to detect such arbitrary structures.